tests/php/StoreApi/Utilities/NoticeHandler.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * NoticeHandler Tests.
+ */
+
+namespace Automattic\WooCommerce\Blocks\Tests\StoreApi\Utilities;
+
+use Automattic\WooCommerce\Blocks\StoreApi\Routes\RouteException;
+use Automattic\WooCommerce\Blocks\StoreApi\Utilities\CartController;
+use Automattic\WooCommerce\Blocks\StoreApi\Utilities\NoticeHandler;
+use PHPUnit\Framework\TestCase;
+use \WC_Helper_Product as ProductHelper;
+
+class NoticeHandlerTests extends TestCase {
+
+	public function test_convert_notices_to_exceptions() {
+		$this->expectException(RouteException::class);
+		$this->expectExceptionMessage('This is an error message with Some HTML in it.');
```

Comment:
The formatting looks off in these two lines:
```suggestion
		$this->expectException( RouteException::class );
		$this->expectExceptionMessage( 'This is an error message with Some HTML in it.' );
```